### PR TITLE
Getting docker logs always to debug issue 8399

### PR DESCRIPTION
**Pull request by soltysh**
I'm forcing to download docker logs to help debug #8399.

@smarterclayton ptal

### CODE REVIEW

File: hack/util.sh [hunk 1]
```diff
@@ -570,10 +570,11 @@ function cleanup_openshift {
 		set -u
 	fi
 
-	if grep -q 'no Docker socket found' "${LOG_DIR}/openshift.log"; then 
+	# TODO soltysh: restore the if back once #8399 is resolved
+	# if grep -q 'no Docker socket found' "${LOG_DIR}/openshift.log"; then
 		# the Docker daemon crashed, we need the logs
-		journalctl --unit docker.service --since -4hours > "${LOG_DIR}/docker.log"
-	fi
+	journalctl --unit docker.service --since -4hours > "${LOG_DIR}/docker.log"
```
Comment (smarterclayton) [inline hack/util.sh]: Slim this down to 15 minutes, that may still be too much logs but we only need the last run.

Comment (smarterclayton) [inline hack/util.sh]: I don't want to blow out jenkins (the docker log file is massive)

**Comment by soltysh** [comment]
@smarterclayton done

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**